Base/Python/slicer/util.py

```diff
@@ -780,13 +780,23 @@ def loadColorTable(filename, returnNode=False):
 
 
 def loadFiberBundle(filename, returnNode=False):
-    """Load node from file.
+    """Load fiber bundle node from file. The ``SlicerDMRI`` needs to be
+    installed to load fiber bundle data.
 
     :param filename: full path of the file to load.
     :param returnNode: Deprecated.
     :return: loaded node (if multiple nodes are loaded then a list of nodes).
       If returnNode is True then a status flag and loaded node are returned.
     """
+
+    emm = slicer.app.extensionsManagerModel()
+    is_ext_installed = emm.isExtensionInstalled("SlicerDMRI")
+    if not is_ext_installed:
+        errorMessage = f"SlicerDMRI extension not installed: failed to read data from file: {filename}"
```

Comment:
```suggestion
    if slicer.app.ioManager().registeredFileReaderCount("FiberBundleFile") == 0:
        errorMessage = f"FiberBundleFile reader not registered: failed to read data from file: {filename}"
        errorMessage += "\n" + "Try to install the SlicerDMRI extension"
        
```

Since extension may either be installed or bundled (e.g in the context of custom application), the test should be more specific.

Reply:
Done in de0d64b.